waspc/data/Generator/templates/react-app/src/auth/pages/OAuthCodeExchange.js

```diff
@@ -0,0 +1,55 @@
+{{={= =}=}}
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+
+import config from '../../config.js'
+import api, { setAuthToken } from '../../api.js'
+
+// This component is rendered in response to an OAuth 2.0 redirect request,
+// where it takes all of the query string parameters and sends them to the Wasp
```

Comment:
Maybe additionally emphasize that this Oauth 2.0 redirect request comes from the oauth provider. Hm, maybe you could very quickly mention the oath flow itself: "After user authenticates via Oauth 2.0 provider, this is the page provider should redirect them to, while providing query string parameters that contain information needed for the API server to authenticate with the provider. This page forwards that information to the API server and in response get a JWT, which it stores on the client, therefore completing the OAuth authentication process."

Reply:
Sure, sounds good, I can update it! 👍🏻